MAiD - Case Management App/force-app/main/default/objects/Case/fields/Reporting_Status__c.field-meta.xml

```diff
@@ -5,10 +5,10 @@
     <formula>IF( AND(MAiD_Reporting_Deadline__c &gt;=  TODAY(),  Pharmacy_Reporting_Deadline__c &gt;= TODAY(), OR(ISBLANK( X1632_Date_Received__c ), ISBLANK(  X1633_Date_Received__c ), ISBLANK(  
         X1634_Date_Received__c ), ISBLANK(RX_MAR_Date_Received__c), ISBLANK(X1641_Date_Received__c ))), &apos;AWAITING&apos;, 
        
-        IF(AND(MAiD_Reporting_Deadline__c &lt;  TODAY(),  Pharmacy_Reporting_Deadline__c &gt; TODAY(), ISBLANK(X1641_Date_Received__c)), &apos;WAITING ON PHARMACY&apos;, 
+        IF(AND(MAiD_Reporting_Deadline__c &lt;  TODAY(),  Pharmacy_Reporting_Deadline__c &gt;= TODAY(), ISBLANK(X1641_Date_Received__c)), &apos;WAITING ON PHARMACY&apos;, 
         IF(AND(MAiD_Reporting_Deadline__c &lt;  TODAY(),  Pharmacy_Reporting_Deadline__c &lt; TODAY(), OR(ISBLANK( X1632_Date_Received__c ), ISBLANK(  X1633_Date_Received__c ), ISBLANK(  X1634_Date_Received__c ), ISBLANK(RX_MAR_Date_Received__c), ISBLANK(X1641_Date_Received__c ))), &apos;OVERDUE&apos;, 
         IF(NOT(OR(ISBLANK( X1632_Date_Received__c ), ISBLANK(  X1633_Date_Received__c ), ISBLANK(  X1634_Date_Received__c ), ISBLANK(RX_MAR_Date_Received__c), ISBLANK(X1641_Date_Received__c ))), 
-        IF(AND(MAiD_Reporting_Deadline__c &gt;= X1632_Date_Received__c, MAiD_Reporting_Deadline__c &gt;= X1633_Date_Received__c, MAiD_Reporting_Deadline__c &gt;= X1634_Date_Received__c , MAiD_Reporting_Deadline__c &gt;= RX_MAR_Date_Received__c, Pharmacy_Reporting_Deadline__c &gt;= X1641_Date_Received__c), &apos;MET REQUIREMENTS&apos;,&apos;DID NOT MEET REQUIREMENTS&apos;), &apos;OVERDUE&apos;
+        IF(AND(MAiD_Reporting_Deadline__c &gt;= X1632_Date_Received__c, MAiD_Reporting_Deadline__c &gt;= X1633_Date_Received__c, MAiD_Reporting_Deadline__c &gt;= X1634_Date_Received__c , MAiD_Reporting_Deadline__c &gt;= RX_MAR_Date_Received__c, Pharmacy_Reporting_Deadline__c &gt;= X1641_Date_Received__c), &apos;MET REQUIREMENTS&apos;,&apos;DID NOT MET REQUIREMENTS&apos;), &apos;OVERDUE&apos;
```

Comment:
The correct term would be "Did not meet requirements" @RanadheerRG It was correct before it was changed. Alternatively it can be "Requirements not met"

Reply:
Hi @cgijeffolsen ,
I have updated the spelling to "Did not Meet Requirements"